sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -374,6 +374,13 @@ def _add_argparse_args(cls, parser):
     parser.add_argument('--template_location',
                         default=None,
                         help='Save job to specified local or GCS location.')
+    parser.add_argument(
+        '--label', '--labels',
+        dest='labels',
+        action='append',
+        default=None,
+        help='Labels that will be applied to the billing records for this job. '
```

Comment:
<!--thread_id:cc_144458602_t; commit:a4e36ef94c4f7a26c96e424c18408b5ae42ca16d; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Is "labels" a general purpose Dataflow concept, or just for billing?

<!--section:body-->
Removed billing.

It reads like a general GCP thing (https://cloud.google.com/resource-manager/docs/using-labels) and shown in the UI similar to pipeline options in a separate are with no mention of billing.

@zongweiz is this accurate?